baseplate/sidecars/__init__.py

```diff
@@ -4,6 +4,10 @@
 from typing import NamedTuple
 from typing import Optional
 
+import gevent
+
+gevent.monkey.patch_all()
```

Comment:
@eaceaser lmk if there's somewhere better to put this

Reply:
we still need to remove this b/c otherwise it'll monkeypatch the sidecar when using the posix queue